github-webhooks-to-s3/firehose.tf

```diff
@@ -0,0 +1,102 @@
+module "bucket" {
+  source = "../aws-s3-private-bucket"
```

Comment:
this path is not valid here, we might also have to opensource this module as well